### DISCUSSION TITLE
Multiple section links behind a single Image

### DISCUSSION THREAD

**Discussion by goldcode** [QuestPDF/QuestPDF#536] [General]
Lets say I have an image with 10 cars in it, would it be possible to click on those cars and then jump to a that 'car section' in the pdf. Wondering if a combination of layers like a canvas layer and image layer would work. the canvas layer would contain some rectangle (car) with a section link? Would this be an approach at all or is there a better way?

**Comment by girlpunk** [comment]
Using a layer to place row/grid cells on top of the image is likely the easiest way to create this, though the canvases would also work.

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
